### DISCUSSION TITLE
Should we enable discussions here?

### DISCUSSION THREAD

**Discussion by yarikoptic** [dandi/dandischema#201] [Polls]
@candleindark  wondered if we should start discussions here to possibly have more interactive etc discussion in addition to issues... let's try

**Comment by candleindark** [comment]
> @ito wondered if we should start discussions here to possibly have more interactive etc discussion in addition to issues... let's try

@yarikoptic, by ito, I think you mean @candleindark?

**Reply by yarikoptic** [reply]
right, corrected!

**Comment by kabilar** [comment]
To make it easier for users, perhaps we should consolidate discussions and create a single discussions board on the [GitHub Org level](https://github.com/dandi) (which we would need to enable)?

**Reply by candleindark** [reply]
I think that can be a good idea. However, if we choose that route, can we separates the discussion posts according to the specific repos that they are pertaining to?

**Reply by kabilar** [reply]
Thanks.  I don't think that there is a built in mechanism, but perhaps we can use `Categories` or add the relevant repo to the title within brackets (e.g. `[dandi-archive] Some discussion about the API`).

**Reply by candleindark** [reply]
That sounds good to me.
